kore/src/Kore/Builtin/Endianness.hs

```diff
@@ -27,7 +27,9 @@ import Data.String
     )
 
 import qualified Kore.Attribute.Symbol as Attribute.Symbol
-import Kore.Builtin.Builtin
+import Kore.Builtin.Builtin hiding
```

Comment:
Not sure how would that look, but, if the list would not bee too long, why not import only what's needed?

The same in Signedness.hs